src/app/pages/MediaAssetPage/index.jsx

```diff
@@ -30,14 +33,19 @@ import {
   getAboutTags,
 } from '#lib/utilities/parseAssetData';
 
+import { RequestContext } from '#contexts/RequestContext';
+
 // Page Handlers
 import withContexts from '#containers/PageHandlers/withContexts';
 import withPageWrapper from '#containers/PageHandlers/withPageWrapper';
 import withError from '#containers/PageHandlers/withError';
 import withLoading from '#containers/PageHandlers/withLoading';
 import withData from '#containers/PageHandlers/withData';
 
+const isLegacyAsset = url => url.split('/').length > 7;
```

Comment:
Would this logic be useful for other pods? Could be worth extracting into its own file?